src/audio-worklet/free-queue/example/index.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
```

Comment:
Let's change this to 2-space indentation. (Google style)

Reply:
Sure!